### PR TITLE
Add skywalking-eyes to check License.

### PR DESCRIPTION
**Pull request by totalo**
For #11876.

### CODE REVIEW

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
```
Comment (terrymanu) [inline .mvn/wrapper/MavenWrapperDownloader.java:9]: I am not sure whether can change license of `MavenWrapperDownloader`, it is not source code of ShardingSphere.
Could you contact skywalking community to confirm it?

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: .mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (totalo) [reply .mvn/wrapper/MavenWrapperDownloader.java:9]: I found from the code of `skywalking` that this generation purchase is also licensed under this license. It seems that it cannot be changed. I will ignore the `.mvn` for now.

Comment (wu-sheng) [reply .mvn/wrapper/MavenWrapperDownloader.java]: Oh, this was on their original header, even it is wrong. But we don't know what should we change to. And all things not updated after that(2+ years ago)

Comment (wu-sheng) [reply .mvn/wrapper/MavenWrapperDownloader.java]: Welcome to fix this on SkyWalking site too. This was deliberate. @totalo

**Comment by wu-sheng** [comment]
One tip, this should be able to replace rat plugin. 

FYI @kezhenxu94 @fgksgf we have progress here.

**Review by terrymanu — APPROVED** [review]

**Comment by terrymanu** [comment]
Thank you for the wonderful job

File: .mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (totalo) [reply .mvn/wrapper/MavenWrapperDownloader.java]: Okay, I will fix this question.

**Pull request merged**